docs-web/app-protect/troubleshooting.md

```diff
@@ -30,6 +30,10 @@ The table below categorizes some potential problems with the Ingress Controller
      - The Ingress Controller NGINX verification timeouts while starting for the first time or while reloading after a change.
      - Check the logs for ``Unable to fetch version: X`` message.
      - Too many Ingress Resources with App Protect enabled. Check the `NGINX fails to start/reload section <#nginx-fails-to-start-or-reload>`_ of the Known Issues. 
+   * - Reload time.
```

Comment:
reload time is already covered by the previous row. Better to extend the Troubleshooting method column with an additional sentence: Check the HTTP response time to resources used in App Protect policy external references. Note that on its own, external references is not the only reason for increased reload. That's why the suggestion.